### PR TITLE
PHP: stop testing against PHP5

### PR DESCRIPTION
**Pull request by HannahShiSFB**
This PR is part of deprecate php5. Working towards #23645

### CODE REVIEW

**Comment by stanley-cheung** [comment]
For the "Basic Tests PHP Linux" failure, it's likely you need to change this line: https://github.com/grpc/grpc/blob/master/tools/internal_ci/linux/pull_request/grpc_basictests_php.cfg#L29 from `php` to `php7`.

**Comment by stanley-cheung** [comment]
For the "Interop Cloud-to-Prod Tests (internal CI)" and "Interop Cloud-to-Cloud Tests (internal CI)", you likely have to fix this file: https://github.com/grpc/grpc/blob/master/tools/run_tests/run_interop_tests.py.

- Probably should remove this whole block: https://github.com/grpc/grpc/blob/master/tools/run_tests/run_interop_tests.py#L486-L514
- And most critically, remove this line: https://github.com/grpc/grpc/blob/master/tools/run_tests/run_interop_tests.py#L737 to stop us from even testing the `php` (which is PHP5) set from the beginning.
- Then remove some other references to `php` in https://github.com/grpc/grpc/blob/master/tools/run_tests/run_interop_tests.py#L882
- Let's remove this block as well: https://github.com/grpc/grpc/blob/master/tools/run_tests/run_interop_tests.py#L1164-L1169

**Comment by HannahShiSFB** [comment]
and https://github.com/grpc/grpc/blob/a14395495d2199c50e1f0dc5e536355798193d8b/tools/interop_matrix/client_matrix.py#L62

**Comment by stanley-cheung** [comment]
> and https://github.com/grpc/grpc/blob/master/tools/interop_matrix/client_matrix.py#L62

Yes remove `php` from this one as well.

**Comment by stanley-cheung** [comment]
I ran the `tools/dockerfile/push_testing_images.sh` script to make sure the copies are pushed to docker hub and re-ran the tests in this PR. Looking good so far.

File: src/php/bin/run_all_docker_images.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@
 set -e
 cd $(dirname $0)/../../..
 
-ALL_IMAGES=( grpc-ext grpc-src alpine centos7 php5 php-src php-future php-zts
+ALL_IMAGES=( grpc-ext grpc-src alpine centos7 php-src php-future php-zts
```
Comment (stanley-cheung) [inline src/php/bin/run_all_docker_images.sh:19]: Can you also delete the dockerfile at `templates/src/php/docker/php5/Dockerfile.template` please?

**Review by stanley-cheung — COMMENTED** [review]
The rest looks good. Thanks!

File: src/php/bin/run_all_docker_images.sh (hunk 1 above)
Comment (HannahShiSFB) [reply src/php/bin/run_all_docker_images.sh:19]: comment addressed

Comment (stanley-cheung) [reply src/php/bin/run_all_docker_images.sh:19]: Oh, and `src/php/docker/php5/Dockerfile` too please.

Comment (HannahShiSFB) [reply src/php/bin/run_all_docker_images.sh:19]: comment addressed

**Review by stanley-cheung — APPROVED** [review]
LGTM. Thanks for iterating on this!

**Pull request merged**